tests/repository/recommendation_queries_test.py

```diff
@@ -129,36 +130,67 @@ def test_deletes_unread_recommendations_older_than_one_week(self, app):
         old_recommendation = create_recommendation(offer, user, date_created=today - timedelta(days=8), date_read=None)
         new_recommendation = create_recommendation(offer, user, date_created=today, date_read=None)
         PcObject.save(old_recommendation, new_recommendation)
+        old_recommendation_id = old_recommendation.id
 
         # When
-        delete_all_unread_recommendations_older_than_one_week()
+        delete_useless_recommendations()
 
         # Then
         recommendations = Recommendation.query.all()
         assert new_recommendation in recommendations
-        assert old_recommendation not in recommendations
+        assert len(recommendations) == 1
+        recommendation_ids = [recommendation.id for recommendation in recommendations]
+        assert old_recommendation_id not in recommendation_ids
+
+    @clean_database
+    def test_deletes_unread_recommendations_older_than_one_week_with_pagination(self, app):
```

Comment:
```suggestion
    def test_deletes_useless_recommendations_with_pagination(self, app):
```
Ton intention dans ce test est de tester la pagination et pas le older thant one week que tu fais plus haut.
De plus, je renommerai tous tes tests avec `useless`` au lieu de `unread` pour être en adéquation avec ta fonction. Qu'en penses tu ?

Reply:
:1